src/osd/modules/netdev/taptun.cpp

```diff
@@ -48,8 +59,14 @@ class netdev_tap : public osd_netdev
 protected:
 	int recv_dev(uint8_t **buf);
 private:
+#if defined(WIN32)
+	HANDLE m_handle;
```

Comment:
Are you initialising `m_handle` to a safe value on construction?  I can't see that happening.  If somehow an exception is thrown before you assign it, you might close an invalid handle.  Probably good to initialise it to `INVALID_HANDLE_VALUE` here.